package.json

```
@@ -30,8 +31,10 @@
     "readable-stream": "^3.4.0"
   },
   "devDependencies": {
+    "@types/readable-stream": "^2.3.13",
     "faucet": "~0.0.1",
     "standard": "^16.0.3",
-    "tape": "^5.2.2"
+    "tape": "^5.2.2",
+    "typescript": "^4.7.3"
```

```suggestion
    "typescript": "~4.7.3"
```

typescript breaks too much between semver-minor releases, especially when you're just consuming it to check JS